src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-wed calendar--month-29">
+      <div class="calendar__day calendar__day--1"></div>
+      <div class="calendar__day calendar__day--2"></div>
+      <div class="calendar__day calendar__day--3"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```
There is no need to add additional modifiers to calendar elements. You can get access to them using pseudoclasses